content/en/docs/concepts/services-networking/_index.md

```diff
@@ -5,37 +5,72 @@ description: >
   Concepts and resources behind networking in Kubernetes.
 ---
 
+[//]: # (Change section to Kubernetes network model and update docs references )
 ## The Kubernetes network model
 
-Every [`Pod`](/docs/concepts/workloads/pods/) in a cluster gets its own unique cluster-wide IP address. 
-This means you do not need to explicitly create links between `Pods` and you 
-almost never need to deal with mapping container ports to host ports.  
-This creates a clean, backwards-compatible model where `Pods` can be treated 
-much like VMs or physical hosts from the perspectives of port allocation, 
-naming, service discovery, [load balancing](/docs/concepts/services-networking/ingress/#load-balancing), 
-application configuration, and migration.
+Every [`Pod`](/docs/concepts/workloads/pods/) in your cluster gets its own unique cluster-wide IP address. This is referred to as the "IP-per-pod" model.
+
+[//]: # (Understand the ports related to TCP/UDP. Don't think host ports is useful for the reader as this point.) 
+[//]: # (This means you do not need to explicitly create links between `Pods` and you
+almost never need to deal with mapping container ports to host ports.)  
+
+This model provides you with the following advantages:
+
+* clean, backwards-compatible model approach where `Pods` can be treated 
+much like VMs or physical hosts. from the perspectives of port allocation, networking,
+* supports naming, service discovery, [load balancing](/docs/concepts/services-networking/ingress/#load-balancing), 
+application configuration and migration.
+  
+This model is not only less complex overall, but it is compatible
+with the desire for Kubernetes to enable low-friction porting of your apps from VMs
+to containers. If your job previously ran in a VM, your VM had an IP and could
+talk to other VMs in your project.  This is the same basic model.
 
 Kubernetes imposes the following fundamental requirements on any networking
 implementation (barring any intentional network segmentation policies):
 
-   * pods can communicate with all other pods on any other [node](/docs/concepts/architecture/nodes/) 
-     without NAT
-   * agents on a node (e.g. system daemons, kubelet) can communicate with all
-     pods on that node
+* containers in same pod can communicate with each other
+* pods can communicate with all other pods on the same or separate [nodes](/docs/concepts/architecture/nodes/)
+without network address translation (NAT)
+* pods can communicate with other pods on same or separate nodes using L2 bridging, native L3 (IPv4, IPv6) networking, or a CNI-specific form of tunnel encapsulation. 
+
+[//]: # (Not sure the agent communications is part of the K8s network model)
```

Comment:
We should show that a Pod running in host network mode can communicate with another Pod in normal cluster network mode, provided that they are both on the same Pod.

That might be something we save for a followup, but it's good to show it one day.

Also that _some_ network architectures allow for this to happen even when the 2 Pods are on different modes, but that Kubernetes doesn't require or guarantee this.